### PR TITLE
Upgrade to VerneMQ version 1.12.6.2 (#23)

### PR DESCRIPTION
**Pull request by baranga**
This upgrades VerneMQ to version 1.12.6.2 and solves #23.

### CODE REVIEW

**Review by alrik — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -31,4 +31,4 @@ COPY \
   /opt/broker/plugins/lib/realmq_adapter
 
 CMD ["/run.sh"]
-EXPOSE 1883 44053 8888 8080
```
Comment (alrik) [inline Dockerfile:34]: **question**: @baranga Don't we also need to export the websocket port `8080`?

**Review by alrik — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (baranga) [reply Dockerfile:34]: We do not necessarily need to expose it via Dockerfile but we should. I'll adjust that.

Comment (baranga) [reply Dockerfile:34]: :heavy_check_mark: Done

File: vernemq.conf.sh [hunk 2]
```diff
@@ -360,6 +455,7 @@ $(tls_keyfile)
 ##
 ## Acceptable values:
 ##   - text
+## listener.ssl.ciphers =
 #listener.ssl.ciphers = ${TLS_CIPHERS}
```
[the remark this answers is not in the document]
Comment (baranga) [reply vernemq.conf.sh:461]: No particular reason. The default (`## listener.ssl.ciphers =`) seems to be slipped in when upgrading the config file.

[the remark this answers is not in the document]
Comment (baranga) [reply vernemq.conf.sh]: I removed it.

**Review by alrik — APPROVED** [review]
:+1:

**Pull request merged**